bin/ch/ch.cpp

```diff
@@ -129,8 +128,8 @@ void __stdcall PrintChakraCoreVersion()
                 // Doesn't matter if you are on 32 bit or 64 bit,
                 // DWORD is always 32 bits, so first two revision numbers
                 // come from dwFileVersionMS, last two come from dwFileVersionLS
-                printf("%s version %d.%d.%d.%d\n",
-                    chakraDllName,
+                wprintf(_u("%s version %d.%d.%d.%d\n"),
+                    GetChakraDllNameW(),
```

Comment:
I thought of that but I read somewhere that there's a bit of disagreement about what %s and %S should mean for wprintf. Apparently on Linux, %S is wide string even in wprintf and %s is ASCII string even in wprintf, but on Windows the meanings are flipped for wprintf. It might be the case that this is not true or the PAL version of wprintf takes care of this so that they behave the same on both platforms. I'll check.